Assets/HoloToolkit-Examples/SpatialUnderstanding/Scripts/UI.cs

```diff
@@ -247,6 +248,7 @@ private void SetupMenus()
             AddButton("OnEdge NearCenter", Panels.LevelSolver, () => { LevelSolver.Instance.Query_OnEdge_NearCenter(); timeLastQuery = DateTime.MinValue; });
             AddButton("OnFloor AwayFromMe", Panels.LevelSolver, () => { LevelSolver.Instance.Query_OnFloor_AwayFromMe(); timeLastQuery = DateTime.MinValue; });
             AddButton("OnFloor NearMe", Panels.LevelSolver, () => { LevelSolver.Instance.Query_OnFloor_NearMe(); timeLastQuery = DateTime.MinValue; });
+            AddButton("Large positions sittable", Panels.Topology, () => { SpaceVisualizer.Instance.Query_Topology_FindLargePositionsSittable(); timeLastQuery = DateTime.MinValue; });
```

Comment:
This creates a duplicate button in the Topology menu. Was in meant to go under `Panels.LevelSolver`, like the other buttons near it in this list?

Reply:
@trzy